ngraph/python/tests/test_onnx/test_ops_matmul.py

```diff
@@ -99,33 +106,20 @@ def import_and_compute_gemm(input_a, input_b, input_c, **kwargs):
     return computation(input_a, input_b, input_c)[0]
 
 
-def test_op_matmul():
-    # vector @ vector
-    data = ([1, 2], [1, 3])
-    assert np.array_equal(import_and_compute_matmul(*data), np.matmul(*data))
-
-    data = ([1, 2, 3], [[4], [5], [6]])
-    assert np.array_equal(import_and_compute_matmul(*data), np.matmul(*data))
-
-    data = ([[1, 2, 3]], [1, 2, 3])
-    assert np.array_equal(import_and_compute_matmul(*data), np.matmul(*data))
-
-    # vector @ matrix
-    data = ([1, 2, 3], [[4, 5], [6, 7], [8, 9]])
-    assert np.array_equal(import_and_compute_matmul(*data), np.matmul(*data))
-
-    # matrix @ vector
-    data = ([[1, 2, 3], [4, 5, 6]], [[7], [8], [9]])
-    assert np.array_equal(import_and_compute_matmul(*data), np.matmul(*data))
-
-    # matrix @ matrix
-    data = ([[1, 2], [3, 4]], [[5, 6], [7, 8]])
-    assert np.array_equal(import_and_compute_matmul(*data), np.matmul(*data))
-
-    data = ([[1, 2, 3], [4, 5, 6]], [[7, 8], [9, 10], [11, 12]])
-    assert np.array_equal(import_and_compute_matmul(*data), np.matmul(*data))
-
-    data = ([[1, 2], [3, 4], [5, 6]], [[7, 8, 9], [10, 11, 12]])
+@pytest.mark.parametrize(
+    "data, description",
+    [
+        pytest.param(([1, 2], [1, 3]), "vector at vector 1", marks=xfail_issue_35916),
```

Comment:
`@` means the matmul operation here, not `at`
```suggestion
        pytest.param(([1, 2], [1, 3]), "vector @ vector 1", marks=xfail_issue_35916),
```

Reply:
changed to "and"